samples/drivers/ht16k33/README.rst

```diff
@@ -0,0 +1,42 @@
+.. _ht16k33:
+
+HT16K33 LED driver with keyscan
+###############################
+
+Overview
+********
+
+This sample controls the LEDs connected to a `Holtek HT16K33`_
+driver. The sample supports any number of LEDs connected to the
+rows/columns of the HT16K33.
+
+The LEDs are controlled using the following pattern:
+
+ 1. turn on all 128 LEDs one-by-one
```

Comment:
Will the sample work if only 5 LEDs are connected?

Reply:
Yes.